roles/database/files/sql/idempotent/fworch-texts.sql

```diff
@@ -5164,6 +5170,14 @@ INSERT INTO txt VALUES ('H5638', 'English', 'Insert text for extra configuration
 INSERT INTO txt VALUES ('H5639', 'German',  'Hier wird der Meldungstext f&uuml;r Fehler im externen Beantragungsprozess eingetragen.');
 INSERT INTO txt VALUES ('H5639', 'English', 'Insert customized text for errors in requests to external ticket system.');
 
+INSERT INTO txt VALUES ('H5640', 'German',  'Auswahl des LDAP-Systems, in dem die Eigent&uuml;mergruppen gepflegt werden.');
```

Comment:
also include texts in HelpSettingsModelling.razor

Reply:
thanks for finding this